composer.json

```diff
@@ -10,7 +10,7 @@
   },
   "require": {
     "php": ">=8.1",
-    "illuminate/http": "^9.0 || ^10.0",
```

Comment:
Should we remove laravel 9 compatibility?

Reply:
I think it doesn't have any impact on the current code, so we can keep it for now.